core/src/main/java/org/hisp/dhis/android/core/legendset/internal/DataElementLegendSetLinkStoreImpl.kt

```diff
@@ -55,4 +57,12 @@ internal class DataElementLegendSetLinkStoreImpl(
             w.bind(3, o.sortOrder())
         }
     }
+
+    override fun getLinksForDataElement(dataElementUid: String): List<ObjectWithUid> {
+        val whereClause = WhereClauseBuilder()
+            .appendKeyStringValue(DataElementLegendSetLinkTableInfo.CHILD_PROJECTION.parentColumn, dataElementUid)
```

Comment:
It would be good to get rid of these `CHILD_PROJECTION` properties if possible. The TableInfo classes will be generated by KSP, so it would be better not to generate these properties. As this function is in a very specific context, I would use the parent column (DataElement) directly and remove the CHILD_PROJECTION property.

We can discuss it in the daily and, if we finally change it, it could be changed in another PR.